com.ibm.wala.core/src/com/ibm/wala/ipa/callgraph/propagation/cfa/ZeroXInstanceKeys.java

```diff
@@ -319,12 +319,14 @@ public static boolean isThrowable(IClass c) {
     return c.getClassHierarchy().isSubclassOf(c, c.getClassHierarchy().lookupClass(TypeReference.JavaLangThrowable));
   }
 
-  public boolean isStackTraceElement(IClass c) {
+  /** BEGIN Custom Change: make this method static (like isThrowable) */
```

Comment:
Do we really need these custom change comments, now that we are merging these changes back?